node/node_newblock.go

```diff
@@ -49,105 +47,90 @@ func (node *Node) WaitForConsensusReadyv2(readySignal chan struct{}, stopChan ch
 						continue
 					}
 
-					coinbase := node.Consensus.SelfAddress
-					if err := node.Worker.UpdateCurrent(coinbase); err != nil {
-						utils.Logger().Error().
-							Err(err).
-							Msg("Failed updating worker's state")
-						continue
-					}
-
-					// Normal tx block consensus
-					selectedTxs := node.getTransactionsForNewBlock(coinbase)
-					if err := node.Worker.UpdateCurrent(coinbase); err != nil {
-						utils.GetLogger().Error("Failed updating worker's state", "Error", err)
-					}
-					utils.Logger().Info().
+					utils.Logger().Debug().
 						Uint64("blockNum", node.Blockchain().CurrentBlock().NumberU64()+1).
-						Int("selectedTxs", len(selectedTxs)).
-						Msg("PROPOSING NEW BLOCK ------------------------------------------------")
+						Msg("=========Proposing New Block==========")
```

Comment:
Can we keep the current message?  This and BINGO/HOORAY are landmark log lines that people may use often for liveness checks.  We should not change these messages without first communicating them out to the users.

Reply:
ok, agreed. Reverted.